tests/dialects/test_func.py

```diff
@@ -91,6 +93,45 @@ def test_callable_constructor():
     assert f.body.ops == []
 
 
+def test_call():
+    # Create two constants and add them, then return
+
+    a = Constant.from_int_and_width(1, i32)
+    b = Constant.from_int_and_width(2, i32)
+    # Operation to add these constants
+    c = Addi.get(a, b)
+    ret0 = Return.get(c)
+
+    # Create a region to include a, b, c and return
+    region = Region.from_operation_list([a, b, c, ret0])
+
+    # Use this region to create a func0
+    func0 = FuncOp.from_region("func0", [a.results[0].typ, b.results[0].typ],
+                               [c.results[0].typ], region)
+
+    # Use these region to create a func0
+    call0 = Call.get(func0.attributes['sym_name'].data, [a, b],
+                     [ret0.operands[0].typ])
```

Comment:
```suggestion
                     [ret0.arguments[0].typ])
```
This doesn't change much, but it makes it more homogeneous with the rest of the code.